sass/directives/_visual-guides.scss

```diff
@@ -125,7 +118,7 @@ $visual-guide-breakpoint-small: 750px;
   }
 
   // The step must be within a <span>, not a <div>, and must contain another <span> for the actual counter
-  span.visual-guide__step {
+  div.visual-guide__step {
```

Comment:
@toastercup The comment and structure was a carryover from the old version of the guide. It was originally written that way due to styles that were being inherited elsewhere. However, because `<div>` is being used now, this no longer applies.

I went ahead and removed the comment and the specificity. This also removed the need to have the extra `<div>` in the container. I'm just able to use `:after` on `.visual-guide__step` and achieve the same results. Thus a developer would only need to include `.visual-guide__step` in the template.

Could you elaborate on the last bit?